spec/02-integration/18-hybrid_rpc/01-rpc_spec.lua

```diff
@@ -69,14 +70,17 @@ for _, strategy in helpers.each_strategy() do
               assert(tonumber(m[2]) >= 9)
 
               -- check the available rpc service
-              table.sort(v.rpc_capabilities)
-              assert.same("kong.sync.v2", v.rpc_capabilities[1])
-              return true
+              for _, c in ipairs(v.rpc_capabilities) do
```

Comment:
We need an RPC call to verify if it actually works

Reply:
This case is `shows DP RPC capability status`, which will not check if the rpc works or not.